### PR TITLE
Add NodeBMCs to locking procedures for locking/unlocking NCNs CSM 1.2

### PR DESCRIPTION
**Pull request by alvarez3-hpe**
## Summary and Scope

This adds instructions for setting the Role of BMCs of NCNs to Management to allow easier locking/unlocking of NodeBMCs of NCNs when locking/unlocking NCNs. Locking NCNs as well as their BMCs will prevent unintended FAS actions from affecting NCNs

## Issues and Related PRs

* Resolves [CASMHMS-5346](https://jira-pro.its.hpecorp.net:8443/browse/CASMHMS-5346)

### Tested on:

  * loki

### Test description:

The new procedure was tested by running it on Loki.

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? N/A
- Were continuous integration tests run? If not, why? N/A
- Was upgrade tested? If not, why? N/A
- Was downgrade tested? If not, why? N/A
- Were new tests (or test issues/Jiras) created for this change? N/A

## Risks and Mitigations

None

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md [hunk 1]
```diff
@@ -110,9 +113,24 @@ Use the `cray hsm locks lock` command to perform locking.
    [Success]
    ComponentIDs = [ "x3000c0s5b0n0", "x3000c0s4b0n0", "x3000c0s7b0n0", "x3000c0s6b0n0", "x3000c0s3b0n0", "x3000c0s2b0n0", "x3000c0s9b0n0", "x3000c0s8b0n0",]
    ```
+   
+1. Lock the NodeBMCs of those management nodes
+   ```bash
+   ncn# cray hsm locks lock create --component-ids $(cray hsm state components list --role management --type node --format json | jq '.Components[].ID' | sed 's/n[0-9]*//;s/"//g' | tr '\n' ',' | sed 's/.$//')
```
Comment (nrockershousen) [inline operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md:119]: Separate command input/output here and in other steps below where it is in one code block. Here is an example:

```
ncn# cray hsm locks lock create --component-ids $(cray hsm state components list --role management --type node --format json | jq '.Components[].ID' | sed 's/n[0-9]*//;s/"//g' | tr '\n' ',' | sed 's/.$//')
```


Example output:


```
Failure = []
[Counts]
Total = 8
Success = 8
Failure = 0
[Success]
ComponentIDs = [ "x3000c0s5b0", "x3000c0s4b0", "x3000c0s7b0", "x3000c0s6b0", "x3000c0s3b0", "x3000c0s2b0", "x3000c0s9b0", "x3000c0s8b0",]
```

File: operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md [hunk 2]
```diff
@@ -88,11 +88,14 @@ The return value of the script is 0 if locking was successful. Otherwise, a non-
...
 
-* To lock all nodes with the _Management_ role.
+***NOTE: When locking NCNs, you must lock their NodeBMCs as well.***
+
+##### To lock all nodes (and their BMCs) with the _Management_ role.
 
    The *processing-model rigid* parameter means that the operation must succeed on all
    target nodes or the entire operation will fail.
 
+1. Lock the management nodes
```
Comment (nrockershousen) [inline operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md:98]: Add a period at the end of all of the steps below.

Comment (alvarez3-hpe) [reply operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md]: Done.

**Comment by alvarez3-hpe** [comment]
Needs modification for marking NodeBMCs of NCNs with the Management role

**Review by nrockershousen — APPROVED** [review]

**Review by nieuwsma — APPROVED** [review]

**Review by nieuwsma — APPROVED** [review]

**Pull request merged**